chat-plugins/info.js

```diff
@@ -109,6 +109,19 @@ var commands = exports.commands = {
 		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
 			buf += '<br />Private rooms: ' + privaterooms;
 		}
+
+		if (user.can('lock') || (user.can('roomban') && Object.keys(room.users).indexOf(targetUser.userid) >= 0)) {
```

Comment:
In my opinion, if the user can't ban the target user then they don't deserve to know that information, nor would it help them in any way.